### ISSUE TITLE
CNNcifar got some problem to work

### ISSUE THREAD

**Issue by eugenewei1997** [AshwinRJ/Federated-Learning-PyTorch#4]
First thanks for the amazing work, but when I want to run CNN on CIFAR10 dataset there is some issue it got runtime error I wonder how to solve it. And the link is the error message.
![issue](https://user-images.githubusercontent.com/50320056/72581047-f32c8180-3918-11ea-9cf3-e48672622fd4.PNG)

Is in update.py line 64

Thanks for you work again.

**Comment by noblevil** [comment]
I have same problem too. if you have solved it, could you tell me your Solution？thanks!

**Comment by eugenewei1997** [comment]
There are two thing sould be revised.
1. In utils.py get_dataset function train_dataset&test_dataset should = datasets.CIFAR10
2. In sampling.py cifar_noniid function  about line 175 labels = np.array(dataset.targets)
And it should be ok to work

**Comment by noblevil** [comment]
> There are two thing sould be revised.
> 
> 1. In utils.py get_dataset function train_dataset&test_dataset should = datasets.CIFAR10
> 2. In sampling.py cifar_noniid function  about line 175 labels = np.array(dataset.targets)
>    And it should be ok to work

谢了
